### PR TITLE
updated README.md setup.py and zrp/about.py

### PR DESCRIPTION
**Pull request by PiotrekZak**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # This call to setup() does all the work
 setup(
     name="zrp",
-    version="0.3.1",
+    version="0.4.0",
```
Comment (kasey-zest) [inline setup.py:21]: version 0.3.2

File: zrp/about.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 __title__ = "zrp"
-__version__ = "0.3.1"
+__version__ = "0.4.0"
```
Comment (kasey-zest) [inline zrp/about.py:2]: version 0.3.2

**Review by kasey-zest — CHANGES_REQUESTED** [review]
need to update versions before moving ahead with the release

File: setup.py (hunk 1 above)
Comment (PiotrekZak) [reply setup.py]: done

File: zrp/about.py (hunk 2 above)
Comment (PiotrekZak) [reply zrp/about.py]: done

**Review by kasey-zest — APPROVED** [review]

**Pull request merged**